website/docs/components/separator/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,30 @@
+
+## Placement
+
+Use separators to distinguish between sections on a page visually. Use them sparingly and purposefully to avoid cluttering the UI and distracting users from the main content.
+
+!!! Do
+
+Use to create a clear break between paragraphs, blocks, or sections.
+
+![Example of a leading icon in the tag](/assets/components/separator/separator-do.png)
```

Comment:
The _alt_ text seems to be not correct (probably copy&pasted the code from another component, better to check all the image instances in this PR)